transactor/src/main/scala/io/mediachain/types/DatastoreTypes.scala

```diff
@@ -0,0 +1,202 @@
+package io.mediachain.types
+
+
+object DatastoreTypes {
```

Comment:
it is a member of a package named io.mediachain.types after all, so types.Datastore and types.Transactor is natural.